skietbaan_fe/src/components/EditGroup.js

```diff
@@ -8,18 +8,18 @@ class EditGroup extends Component {
     this.state = {
       posts: [],
       newArray: [],
-      filterText: ""
+      filterText: "",
+      count:0
     };
     this.toggleHighlight = this.toggleHighlight.bind(this);
     this.handleOnClick = this.handleOnClick.bind(this);
     this.onBack = this.onBack.bind(this);
     this.onChange = this.onChange.bind(this);
   }
   componentWillMount() {
-    fetch("http://localhost:63474/api/Groups/edit?id=" + this.props.id)
+    fetch("http://localhost:64444/api/Groups/edit?id=" + this.props.id)
```

Comment:
No Localhost variables
This makes me also think you missed some base url request please make sure to get all of them

Reply:
thanks i have made the changes